OpenStack-accounting/sql/nova_get_accounting_data.sql

```diff
@@ -1,4 +1,4 @@
-CREATE PROCEDURE `get_accounting_data`(IN starttime datetime, IN endtime datetime)
+CREATE DEFINER=`accounting-db`@`host-172-16-101-160.nubes.stfc.ac.uk` PROCEDURE `get_accounting_data`(IN starttime datetime, IN endtime datetime)
```

Comment:
Is the `CREATE DEFINER` needed for getting accounting data from the other OpenStack components as well?

Reply:
It isnt and is something added in by the mysql workbench that I missed removing from this one.

Reply:
Fixed